app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -270,6 +270,34 @@ function handleKeyUp(event) {
   handleSizeChange(event);
 }
 
+function handleKeyDown(event) {
+    // If Ctrl+Enter OR Command+Enter is pressed
+    if((event.ctrlKey || event.metaKey) && event.keyCode === 13)
```

Comment:
May be we can add a const for example here with the value `13` in order to "auto explain" a bit what this number is, for example `enterKeyCode`. What do you think?